java/1.2/modules/ROOT/pages/configure-java-module-acb.adoc

```diff
@@ -0,0 +1,306 @@
+= Configure the Java Module Using Anypoint Code Builder
+
+Use the Java module in your Anypoint Code Builder project to define custom Java code. To use Anypoint Code Builder to configure the Java module, create a basic integration project in Anypoint Code Builder, add connectors and other components to your Mule application to process your business logic, and then configure the attributes for the Java Module. 
+
+== Before You Begin
+
+Before creating an Anypoint Code Builder integration project, you must:
+
+* xref:anypoint-code-builder::start-acb.adoc[Set up and access the Anypoint Code Builder web or desktop IDE]
+* Have credentials to access Anypoint Platform
+* Ensure that the Visual Studio Code Extension Pack for Java is installed.
+
+[[creating-new-integration-project]]
+== Creating a New Integration Project
+
+include::connectors::partial$configure-acb-connectors-partial.adoc[tag="acb-create-integration-project"]
+
+When you create a new Anypoint Code Builder project, `src/main/java` and `/src/test/java` folders are created. For existing projects, you can manually create this folder. 
+
+
+
+[[add-components]]
+== Add Components to Your Project
+
+Add components to your integration project to build a flow:
+
+. In the *Explorer* view, open the XML file for your project if it's not already open.
+. Select *Build a Flow* from the start card to create an empty flow:
```

Comment:
"start card" is new terminology from an ACB perspective. We call that area of the IDE "the canvas" or "the canvas UI".

Reply:
```suggestion
. Select *Build a Flow* from the start card in the canvas to create an empty flow:
```

Reply:
or more consistent with existing doc
```suggestion
. Select *Build a Flow* from the canvas to create an empty flow:
```